underfs/swift/src/main/java/tachyon/underfs/swift/SwiftUnderFileSystem.java

```diff
@@ -15,51 +15,370 @@
+  private final String mContainerPrefix;
+  /** JOSS access object */
+  private final Access mAccess;
+
+  public SwiftUnderFileSystem(String containerName,
+      TachyonConf tachyonConf) throws Exception {
+    super(tachyonConf);
+    LOG.debug("Constructor init: {}", containerName);
+    AccountConfig config = new AccountConfig();
+    config.setUsername(tachyonConf.get(Constants.SWIFT_USER_KEY));
+    config.setTenantName(tachyonConf.get(Constants.SWIFT_TENANT_KEY));
+    config.setPassword(tachyonConf.get(Constants.SWIFT_API_KEY));
+    config.setAuthUrl(tachyonConf.get(Constants.SWIFT_AUTH_URL_KEY));
+    String authMethod = tachyonConf.get(Constants.SWIFT_AUTH_METHOD_KEY);
+    if (authMethod != null && authMethod.equals("keystone")) {
+      config.setAuthenticationMethod(AuthenticationMethod.KEYSTONE);
+    } else {
+      config.setAuthenticationMethod(AuthenticationMethod.TEMPAUTH);
+    }
+
+    ObjectMapper mapper = new ObjectMapper();
+    mapper.configure(SerializationConfig.Feature.WRAP_ROOT_VALUE, true);
+    mContainerName = containerName;
+    mAccount = new AccountFactory(config).createAccount();
+    mAccess = mAccount.authenticate();
+    Container containerObj = mAccount.getContainer(containerName);
+    if (!containerObj.exists()) {
+      containerObj.create();
+    }
+    mContainerPrefix = Constants.HEADER_SWIFT + mContainerName + PATH_SEPARATOR;
   }
 
   @Override
-  public UnderFSType getUnderFSType() {
-    return UnderFSType.SWIFT;
+  public void close() throws IOException {
+    LOG.debug("close");
+  }
+
+  @Override
+  public void connectFromMaster(TachyonConf conf, String hostname) {
+    LOG.debug("connect from master");
+  }
+
+  @Override
+  public void connectFromWorker(TachyonConf conf, String hostname) {
+    LOG.debug("connect from worker");
+  }
+
+  @Override
+  public OutputStream create(String path) throws IOException {
+    LOG.debug("Create method: {}", path);
+    String newPath = path.substring(Constants.HEADER_SWIFT.length());
+    SwiftOutputStream out = SwiftDirectClient.PUT(mAccess, newPath);
+    return out;
+  }
+
+  @Override
+  public OutputStream create(String path,
+      int blockSizeByte) throws IOException {
+    LOG.warn("Create with block size is not supported"
+        + "with SwiftDirectUnderFileSystem. Block size will be ignored.");
+    return create(path);
+  }
+
+  @Override
+  public OutputStream create(String path, short replication, int blockSizeByte) throws IOException {
+    LOG.warn("Create with block size and replication is not"
+        + "supported with SwiftDirectUnderFileSystem."
+        + " Block size and replication will be ignored.");
+    return create(path);
+  }
+
+  /* @inheritDoc
+   * @see tachyon.underfs.UnderFileSystem#delete(java.lang.String, boolean).
+   * recursive will delete all objects with given prefix.
+   * parent will not be deleted.
+   * Method always returns {@code true}.
```

Comment:
Path may not exists if it's recursive.. for example
if we  have objects inside container: 
a/b1/c1
a/b1/c11
a/b2/c2

There recursive delete a/ will delete all, but "a" doesn't exists